session/pingpong/accountant_caller.go

```diff
@@ -96,7 +99,16 @@ func (ac *AccountantCaller) GetConsumerData(id string) (ConsumerData, error) {
 	}
 	var resp ConsumerData
 	err = ac.doRequest(req, &resp)
-	return resp, errors.Wrap(err, "could not request consumer data accountant")
+	if err != nil {
+		return ConsumerData{}, errors.Wrap(err, "could not request consumer data from accountant")
```

Comment:
You can use `fmt.Errorf("could not request consumer data from accountant: %w", err)`